Added intervals to Python scripts

koerners opened the pull request.

koerners committed:
Interval for graph

koerners committed:
Interval for plot

jbtrystram commented:
commit 17b7094e222662ab9db11aac206a90871f7a6f8f seems fine for adding intervals. 
the other commit add a lot of things, and seems to break things (such as automated multi-plot according to the number of packages). Can you explain what i does ? Or maybe do a separate script ?

koerners committed:
Interval Graph (This time correctly)

koerners committed:
Interval for Data Extraction

Also added if-query to make sure nothing breaks if interval is set to lower than 20 as in that case the progressSincelastInterval doesn't seem to work

koerners commented:
> commit [17b7094](https://github.com/jbtrystram/p2p-iot-simulator/commit/17b7094e222662ab9db11aac206a90871f7a6f8f) seems fine for adding intervals.
> the other commit add a lot of things, and seems to break things (such as automated multi-plot according to the number of packages). Can you explain what i does ? Or maybe do a separate script ?

My mistake should work now :)

jbtrystram commented:
thanks !
